morph_server_base.json

```diff
@@ -10,10 +10,10 @@
   "dependencies": {
     "@esy-ocaml/reason": "*",
     "@opam/dune": "*",
-    "@opam/lwt": "^4.0.0",
+    "@opam/lwt": ">=4.0.0",
```

Comment:
since `morph` depends on Lwt already I think this is redundant?

Reply:
This should be removed, I just did a blanket-change on all files